tests/modules/NotificationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ *
+ * NotificationTest.php is unit test file for Notification module (modules/Notification.php).
+ *
+ */
+namespace Tests\FirstAide;
+$APPLICATION_DIR = str_replace('tests/modules', '', dirname(__FILE__));
+require_once $APPLICATION_DIR.'/includes/settings.php';
+
+/**
+ * Importing required modules/wrappers
+ */
+use FirstAide\Notification;
+use PHPUnit\Framework\TestCase;
+
+class NotificationTest extends TestCase
```

Comment:
100% test coverage indicates every branch is tested in a If/else block, every line of code is tested, and every method is tested.